indico/modules/events/registration/controllers/management/tickets.py

```diff
@@ -64,12 +64,11 @@ def _process(self):
             "event_id": self.event.id,
             "title": self.event.title,
             "date": self.event.start_dt.isoformat(),
-            "version": 1,
+            "version": 2,
             "server": {
                 "base_url": config.BASE_URL,
                 "consumer_key": checkin_app.client_id,
```

Comment:
What about renaming this to `client_id` since "consumer key" is oauth1 terminology?

Reply:
while at it, can we please `s/"/'/` in this whole block? :)